scripts/perftune.py

```diff
@@ -1387,7 +1399,7 @@ def dump_config(prog_args):
         prog_options['mode'] = prog_args.mode
 
     if prog_args.nic:
-        prog_options['nic'] = prog_args.nic
+        prog_options['nic'] = prog_args.nics
```

Comment:
And what about `parse_options_file()`? It should adjusted too.
The default case handling needs to be changed too: `args.nics` should be set to [ 'eth0' ] and the `args.nic` initialization needs to be killed.